aqueductcore/frontend/src/API/graphql/queries/getAllExtentions.tsx

```diff
@@ -0,0 +1,31 @@
+import { gql, QueryHookOptions, useQuery } from "@apollo/client";
+import { GET_ALL_EXTENSIONS_TYPE } from "types/globalTypes";
+
+export const GET_ALL_EXTENTIONS = gql`
+  query getAllPlugins {
```

Comment:
```suggestion
  query getAllExtensions {
```

I hope this well be merged soon: https://github.com/AqueductHub/aqueductcore/blob/a356e0c8f5ea11eef572cd88ee8d5a245424a4df/aqueductcore/backend/routers/graphql/query_schema.py#L107